### PR TITLE
Added translations in validator for Serbian Latin

### PR DESCRIPTION
**Pull request by KristijanKanalas**
[Validator] Added translations
Related to #30188

Working on Serbian Cyrillic I saw that Latin translations are also missing some translations so this is a PR related to that. As said in the Cyrillic PR this should be able to be merged in all branches above 3.4, but I am not sure how to do that.

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf [hunk 1]
```diff
@@ -334,6 +334,38 @@
...
+                <source>This value should be either negative or zero.</source>
+                <target>Ova vrednost bi trebala biti pozitivna ili nula.</target>
+            </trans-unit>
+            <trans-unit id="92">
+                <source>This value is not a valid timezone.</source>
+                <target>Ova vrednost nije validna vremenska zona.</target>
+            </trans-unit>
+            <trans-unit id="93">
+                <source>This password has been leaked in a data breach, it must not be used. Please use another password.</source>
+                <target>Ova lozinka je procurela prilikom prethodnih napada. Nemojte je koristiti.</target>
```
Comment (kunicmarko20) [inline src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf]: `procurela` nekako mi ne zvuci ok a i `Please use another password` nisi preveo

Comment (KristijanKanalas) [reply src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf]: Da u vezi `procurela` pokušao sam na više načina da prevedem ali je problem prevesti ali mi je ovo imalo najviše smisla. Imaš li možda neku drugu ideju?

Comment (KristijanKanalas) [reply src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf]: Sad ću ispraviti ostatak prevoda.

Comment (KristijanKanalas) [reply src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf]: Možda ovako `Ova lozinka je kompromitovana u prethodnim napadima...`?

Comment (kunicmarko20) [reply src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf]: to zvuci mnogo bolje

Comment (KristijanKanalas) [reply src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf]: Done :)

**Review by kunicmarko20 — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @KristijanKanalas.

**Pull request merged**